src/maplayer.c

```diff
@@ -1411,6 +1418,8 @@ int msLayerGetFeatureStyle(mapObj *map, layerObj *layer, classObj *c,
              strncasecmp(stylestring, "symbol", 6) == 0 ||
              strncasecmp(stylestring, "label", 5) == 0) {
     msOGRUpdateStyleFromString(map, layer, c, stylestring);
+  } else if (strcasestr(stylestring, "StyledLayerDescriptor>") != NULL) {
```

Comment:
what about patterns like ``<StyledLayerDescriptor version='1.1.0'>`` ?

Reply:
This is to try and find the closing tag of the SLD e.g. `</StyledLayerDescriptor>` or if namespaces are used `</sld:StyledLayerDescriptor>`

Reply:
ah good point. A comment in the code would help clarify that for future readers

Reply:
Added a comment